### ISSUE TITLE
Embeds not compatible with Pycord master branch

### ISSUE THREAD

**Issue by tibue99** [mahtoid/DiscordChatExporterPy#82]
### DiscordChatExporterPy Version

2.5.3

### Discord Version

py-cord-2.4.1.dev76+g7a6a42c5

### Bug Description

Pycord added new parameters (`author`, `footer`, `image`, `thumbnail`) to `discord.Embed` in [#1996](https://github.com/Pycord-Development/pycord/pull/1996).

This is currently only in their master branch, but this update breaks embeds for chat exporter.

### Bug Traceback

```
Traceback (most recent call last):
  File "C:\Users\Timo\AppData\Local\Programs\Python\Python310\lib\site-packages\chat_exporter\construct\transcript.py", line 166, in export
    return await super().build_transcript()
  File "C:\Users\Timo\AppData\Local\Programs\Python\Python310\lib\site-packages\chat_exporter\construct\transcript.py", line 55, in build_transcript
    message_html, meta_data = await gather_messages(
  File "C:\Users\Timo\AppData\Local\Programs\Python\Python310\lib\site-packages\chat_exporter\construct\message.py", line 389, in gather_messages
    content_html, meta_data = await MessageConstruct(
  File "C:\Users\Timo\AppData\Local\Programs\Python\Python310\lib\site-packages\chat_exporter\construct\message.py", line 83, in construct_message
    await self.build_message()
  File "C:\Users\Timo\AppData\Local\Programs\Python\Python310\lib\site-packages\chat_exporter\construct\message.py", line 91, in build_message
    await self.build_assets()
  File "C:\Users\Timo\AppData\Local\Programs\Python\Python310\lib\site-packages\chat_exporter\construct\message.py", line 226, in build_assets
    self.embeds += await Embed(e, self.guild).flow()
  File "C:\Users\Timo\AppData\Local\Programs\Python\Python310\lib\site-packages\chat_exporter\construct\assets\embed.py", line 57, in flow
    await self.build_author()
  File "C:\Users\Timo\AppData\Local\Programs\Python\Python310\lib\site-packages\chat_exporter\construct\assets\embed.py", line 109, in build_author
    self.author = html.escape(self.embed.author.name) if self.embed.author.name != self.check_against else ""
  File "C:\Users\Timo\AppData\Local\Programs\Python\Python310\lib\site-packages\discord\embeds.py", line 704, in author
    return EmbedAuthor(**getattr(self, "_author", {}))  # type: ignore
TypeError: EmbedAuthor.__init__() missing 1 required positional argument: 'name'
Please send a screenshot of the above error to https://www.github.com/mahtoid/DiscordChatExporterPy
```

**Comment by mahtoid** [comment]
Hello @tibue99,

Thank you for letting me know. Do you know when they plan to release this, just so I can schedule a release around the same time including the fix?

**Comment by tibue99** [comment]
Hey, thank you for taking care of this.

Pycord devs said they want to push for a release "very soon", so I'd guess they'll release it within the next 2 weeks, perhaps even sooner.

**Comment by mahtoid** [comment]
Just to update, this seems to be an issue on Pycords end. I have dropped an issue on their Github and will follow through with what they say there.

**Comment by mahtoid** [comment]
Closing this out, solved this in the `2.6.0` branch using Pycords latest solution. Will get a release out soon.

**Comment by Lulalaby** [comment]
perfect
